src/styles/card.scss

```
@@ -0,0 +1,26 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 0;
+  width: 200px;
+  height: 408px;
+  padding-top: 32px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin: 0 19px;
```

try to use it for pass tests
```suggestion
    margin: 0 auto;
```